### PR TITLE
Add an option to specify output file name

### PR DESCRIPTION
**Pull request by jorgectf**
Added `--option/-o` to specify output's file name.

### CODE REVIEW

File: cli.go [hunk 1]
```diff
@@ -388,6 +388,10 @@ func main() {
 						Name:  "lang, l",
 						Usage: "Filter github repos by language.",
 					},
+					&cli.StringFlag{
+						Name:  "output, o",
+						Usage: "File name to write repositories to.",
```
Comment (gagliardetto) [inline cli.go]:
```suggestion
						Usage: "Filepath to which save the list of target repositories.",
```

File: cli.go [hunk 2]
```diff
@@ -500,6 +504,10 @@ func main() {
 						Name:  "force, y",
 						Usage: "Don't ask for confirmation.",
 					},
+					&cli.StringFlag{
+						Name:  "output, o",
+						Usage: "File name to write repositories to.",
```
Comment (gagliardetto) [inline cli.go]:
```suggestion
						Usage: "Filepath to which save the list of target repositories.",
```

File: cli.go [hunk 3]
```diff
@@ -604,6 +612,10 @@ func main() {
...
```
Comment (gagliardetto) [inline cli.go]:
```suggestion
						Usage: "Filepath to which save the list of target repositories.",
```

File: cli.go [hunk 4]
```diff
@@ -697,6 +709,10 @@ func main() {
...
```
Comment (gagliardetto) [inline cli.go]:
```suggestion
						Usage: "Filepath to which save the list of target repositories.",
```

File: cli.go [hunk 5]
```diff
@@ -2126,9 +2151,9 @@ func saveTargetListToTempFile(cmdName string, targets []string) {
 		log.Fatal(err)
 	}
 
-	fmt.Println(Sf(PurpleBG("Wrote compiled list of targets to temp file %s"), tmpfile.Name()))
+	fmt.Println(Sf(PurpleBG("Wrote compiled list of targets to temp file %s"), outputFileName))
```
Comment (gagliardetto) [inline cli.go]:
```suggestion
	fmt.Println(Sf(PurpleBG("Wrote compiled list of targets to %s"), outputFileName))
```

**Review by gagliardetto — CHANGES_REQUESTED** [review]

**Comment by gagliardetto** [comment]
Awesome! Thanks

**Comment by jorgectf** [comment]
> Awesome! Thanks

Thanks for the merge! 😊

**Pull request merged**